tokio/src/macros/cfg.rs

```diff
@@ -363,3 +363,23 @@ macro_rules! cfg_unstable {
         )*
     }
 }
+
+macro_rules! cfg_coop {
+    ($($item:item)*) => {
+        $(
+            #[cfg(any(
+                    feature = "blocking",
+                    feature = "dns",
+                    feature = "fs",
+                    feature = "io-driver",
+                    feature = "io-std",
+                    feature = "process",
+                    feature = "rt-core",
+                    feature = "sync",
+                    feature = "stream",
+                    feature = "time"
+                    ))]
+            $item
```

Comment:
This is a bit unfortunate. Odds are we probably should just silence warnings for coop, but doing that makes it easy to miss dead code during refactors...